CMakeLists.txt

```diff
@@ -214,17 +242,31 @@ if(BUILD_DX11 AND WIN32)
         file(TO_CMAKE_PATH ${COMPILED_SHADERS} COMPILED_SHADERS)
     endif()
 
-    list(APPEND LIBRARY_SOURCES ${COMPILED_SHADERS}/BC6HEncode_EncodeBlockCS.inc)
-
     if(NOT USE_PREBUILT_SHADERS)
-        add_custom_command(
-            OUTPUT "${COMPILED_SHADERS}/BC6HEncode_EncodeBlockCS.inc"
-            MAIN_DEPENDENCY "${PROJECT_SOURCE_DIR}/DirectXTex/Shaders/CompileShaders.cmd"
-            DEPENDS ${SHADER_SOURCES}
-            COMMENT "Generating HLSL shaders..."
-            COMMAND ${CMAKE_COMMAND} -E env CompileShadersOutput="${COMPILED_SHADERS}" CompileShaders.cmd > "${COMPILED_SHADERS}/compileshaders.log"
-            WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/DirectXTex/Shaders"
-            USES_TERMINAL)
+	    if(MINGW)
+		    file(COPY ${PROJECT_SOURCE_DIR}/DirectXTex/Shaders/BC7Encode.hlsl DESTINATION ${COMPILED_SHADERS})
```

Comment:
Why copy the shader source files instead of using them in place?

Reply:
fxc gave a sigmentat fault when I tried using them in place.

Reply:
I also think the .cmd script was depending upon being in the same dir as the source directory..  That is not the case with the CMakeLists.txt file so we needed to copy the source-file to get things to work properly since the source .hlsl is a dependency for the target compiled .inc file.  I hope I'm making sense.  In any case, I see this as a minor point  that may not be working "fixing".